Sources/Migration/DataPipelineMigrationAssistant.swift

```diff
@@ -1,54 +1,49 @@
-import CioDataPipelines
 import CioInternalCommon
 import Foundation
 
-public protocol DataPipelineMigration: AutoMockable {
-    func handleAlreadyIdentifiedMigratedUser()
-    func handleQueueBacklog()
```

Comment:
I like the simplicity of this protocol with 2 functions. Do we need to replace this protocol with all of the `processX` functions?

Reply:
We do not need this protocol now as we only need to call one method i.e. `performMigration` that will tackle all the migration work.